app/components/support_interface/reference_with_feedback_component.rb

```diff
@@ -11,9 +11,10 @@ class ReferenceWithFeedbackComponent < ViewComponent::Base
              :consent_to_be_contacted,
              to: :reference
 
-    def initialize(reference:, reference_number:)
+    def initialize(reference:, reference_number:, editable: true)
```

Comment:
I can't see where `editable` is being used ... 🤔

Reply:
It isn't called anywhere but without these changes we don't see the "change" links in the support UI. It's a bit of magic that I don't understand and would be very grateful if someone could explain! ✨🧙🏽‍♂️